cybsi/api/dictionary/api.py

```diff
@@ -377,8 +378,9 @@ class DictItemAttributeValue(JsonObjectForm):
 
     Args:
        key: Dictionary item key. The key should consist of characters
-            according to the pattern `[a-zA-Z0-9_ -]` and
-            have length in the range `[1;30]`.
+            according to the pattern `[a-zA-Z0-9_ -:.@`] and
```

Comment:
Здесь тоже паттерн надо поправить
```suggestion
            according to the pattern `[a-zA-Z0-9_ :.@-]` and
```